.github/workflows/docker-image.yml

```diff
@@ -511,6 +511,62 @@ jobs:
       - name: ensure httpd.service is running
         run:  sudo systemctl is-active --quiet httpd.service
 
+  systemd-user-enable-validate:
+    runs-on: ubuntu-latest
+    needs: [ build , build-systemd-amd ]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: enable podman.socket
+        run: |
+          set -x
+          loginctl enable-linger runner
+          sleep 1
+          ls -al /run/user/$UID
+          XDG_RUNTIME_DIR=/run/user/$UID systemctl --user enable --now podman.socket
+
+      - name: pull artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: fetchit-image
+          path: /tmp
+
+      - name: Load the image
+        run: podman load -i /tmp/fetchit.tar
+
+      - name: tag the image
+        run: podman tag quay.io/fetchit/fetchit-amd:latest quay.io/fetchit/fetchit:latest
+
+      - name: pull systemd amd artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: systemd-image-amd
+          path: /tmp
+
+      - name: Load the systemd image
+        run: podman load -i /tmp/systemd-amd.tar
+
+      - name: tag the image
+        run: podman tag quay.io/fetchit/fetchit-systemd-amd:latest quay.io/fetchit/fetchit-systemd:latest
+
+      - name: Start fetchit
+        run: podman run -d --name fetchit -v fetchit-volume:/opt -v ./examples/systemd-enable-user.yaml:/opt/mount/config.yaml -v /run/user/"${UID}"/podman/podman.sock:/run/podman/podman.sock -e XDG_RUNTIME_DIR="/run/user/${UID}" -e HOME="${HOME}" quay.io/fetchit/fetchit-amd:latest
+
+      - name: Wait for fetchit to deploy
+        run: sleep 2m
+
+      - name: Logs
+        run: podman logs fetchit
+
+      - name: list files
+        run: ls ~/.config/systemd/user/httpd.service
+
+      - name: ensure systemd containers are gone
+        run: podman ps -a
+
+      - name: ensure httpd.service is running
+        run:  XDG_RUNTIME_DIR=/run/user/$UID systemctl --user is-active --quiet httpd.service
```

Comment:
I think we may need to verify this check. When I run it on my system without any sort of action it exits cleanly without an error but the service actually isn't running. I tested like this 

```
systemctl --user is-active --quiet httpd.service >/dev/null 2>&1 && echo "Service is active" || exit 1
```

Reply:
oh that is very possible - the systemd pod always exits cleanly, regardless of how the script exits - we need to update that - there must be a way to catch the exit code of the systemd script and the pod should exit with that

Reply:
I'll update the check and also the check for the other sysd enable workflow

Reply:
updated! good catch,thanks :relaxed:

Reply:
and _phew_ tests are still passing so that's good